T/TestU01/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "TestU01"
+version = v"1.2.3-1"
```

Comment:
Please stick to `X.Y.Z`

Reply:
@staticfloat related: https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/1223 :slightly_smiling_face:

Reply:
I was following the version of the source code (which is 1.2.3). Wasn't that policy at some point? Did people get frustrated by all the uncontrolled JLL updates? Since we might make several versions of the wrappers here, we shouldn't use `1.2.3` if we can't append a postfix. Let's then just make it `1.0.0`.